src/SortProgram.java

```diff
@@ -1,56 +1,77 @@
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Scanner;
 
 public class SortProgram {
 
-    private Scanner myScanner = new Scanner(System.in);
+    private ArrayList<String> myList = new ArrayList<>();
 
     public void startProgram() {
-        ArrayList myList = new ArrayList();
-        addToList(myList);
-        System.out.println(myList+"\n");
-        do {
-            printBitch("[ADD]\n[REMOVE]\n");
-            String menuInput = myScanner.next();
-            if (menuInput.equals("ADD")) {
-                printBitch("What would you like to add, bitch?");
-                String item = myScanner.next();
-                myList.add(item);
-                printBitch(myList);
-            } else if (menuInput.equals("REMOVE")) {
-                printBitch("What would you like to remove, bitch?");
-                String item = myScanner.next();
-                int idx = 0;
-                while (idx < myList.size()) {
-                    if (myList.get(idx) == item) {
-                        myList.remove(idx);
-                    } else {
-                        ++idx;
-                    }
-                }
-                printBitch(myList);
-            } else {
+        printBitch("\nHello, bitch, gimme something to list!\n");
+        addListItem();
+        displayMenu();
+    }
+
+    private void displayMenu() {
+        System.out.println("\nYour list: " + myList);
+        printBitch("\n[A]DD\n[R]EMOVE\n[S]ORT\n[E]XIT");
+        menuSelection(getUserInput());
+    }
+
```

Comment:
You'll notice all `switch` cases, _except_ `case "e"`, in the `menuSelection(String input)` method call `displayMenu();` at the end. This is what creates the program loop, which will only end if `menuSelection(String input)` receives a valid "exit" input.